src/Atmos/Model/turbulence.jl

```diff
@@ -0,0 +1,138 @@
+#### Turbulence closures
+abstract type TurbulenceClosure
+end
+
+struct ConstantViscosity <: TurbulenceClosure
+  μ::Float64
+end
+
+vars_gradtransform(::ConstantViscosity) = (:u, :v, :w)
+function gradtransform!(m::ConstantViscosity, transformstate::State, state::State, auxstate::State, t::Real)
+    ρinv = 1 / state.ρ
+    transformstate.u = ρinv * state.ρu
+    transformstate.v = ρinv * state.ρv
+    transformstate.w = ρinv * state.ρw
+end
+  
+vars_diffusive(::ConstantViscosity) = (:τ11, :τ22, :τ33, :τ12, :τ13, :τ23)
+function diffusive!(m::ConstantViscosity, diffusive::State, ∇transform::Grad, state::State, auxstate::State, t::Real)
+  T = eltype(diffusive)
+  μ = T(m.μ)
+  
+  dudx, dudy, dudz = ∇transform.u
+  dvdx, dvdy, dvdz = ∇transform.v
+  dwdx, dwdy, dwdz = ∇transform.w
+
+  # strains
+  ϵ11 = dudx
+  ϵ22 = dvdy
+  ϵ33 = dwdz
+  ϵ12 = (dudy + dvdx) / 2
+  ϵ13 = (dudz + dwdx) / 2
+  ϵ23 = (dvdz + dwdy) / 2
+
+  # deviatoric stresses
+  diffusive.τ11 = 2μ * (ϵ11 - (ϵ11 + ϵ22 + ϵ33) / 3)
+  diffusive.τ22 = 2μ * (ϵ22 - (ϵ11 + ϵ22 + ϵ33) / 3)
+  diffusive.τ33 = 2μ * (ϵ33 - (ϵ11 + ϵ22 + ϵ33) / 3)
+  diffusive.τ12 = 2μ * ϵ12
+  diffusive.τ13 = 2μ * ϵ13
+  diffusive.τ23 = 2μ * ϵ23
```

Comment:
We should include thermal diffusion here as well, as a `Prandtl_number` parameter.